### ISSUE TITLE
Don't cancel file transfer on file manager window closing

### ISSUE THREAD

**Issue by spotlesscoder** [elementary/files#1559]
Just recognized that the file transfer stops when I close the main filemanager window?

WTF?

**Comment by jeremypw** [comment]
If confirmed this is a regression - ongoing file operations are supposed to inhibit closing the app.

**Comment by jeremypw** [comment]
@CodingSpiderFox Please confirm which version of elementary/Files you are running.

**Comment by jeremypw** [comment]
Hmm, looks like Application.quit () ignores the hold count (which is incremented when an operation is ongoing).  Will have to interrupt that signal to prevent this.

**Comment by jeremypw** [comment]
Just tried this on elementary 6.0.  The file operation (copying a large file) continued even when the main window was closed (using the delete button). The progress window remained open and active.

**Comment by CodingSpiderFox** [comment]
It could be that this is already fixed as I use my own fork of files (which
I can't patch since development has switched to focus on eOS 6.0)

Anyways, I'm using Hera 5.1.7

**Comment by hanaral** [comment]
> (which I can't patch since development has switched to focus on eOS 6.0) Anyways, I'm using Hera 5.1.7

You should be able to download the last releases source archive and compile it from there. Could you test if it happens with that version of Files?

**Comment by CodingSpiderFox** [comment]
Last time i tried (which is a couple of weeks ago) there was a dependency missing which is not available on eOS 5.1.7 (I think it was libhandy if I remember correctly)

**Comment by hanaral** [comment]
Are you sure you didn't download the repo as an archive? The latest release tarball is from August last year and didn't yet have libhandy. The tagged releases are accessible from the right sidebar on the repo btw.

**Comment by CodingSpiderFox** [comment]
You mean this ZIP I should use? 

https://github.com/elementary/files/archive/4.5.0.zip

**Comment by hanaral** [comment]
>You mean this ZIP I should use?  ...

@CodingSpiderFox 

If it’s from august then yes

**Comment by CodingSpiderFox** [comment]
Yes, I think it was from August 23rd. I think I won't find the time to reproduce the issue in the near future, as I don't need to copy around my backups anymore at the moment

**Comment by jeremypw** [comment]
Closing as this was not reproduced on later versions
